src/main/java/frc/robot/commands/Autos.java

```diff
@@ -32,6 +32,36 @@ public static CommandBase DriveSlow(DriveSystem driveSubsystem)
   {
     return driveSubsystem.driveDistance(Constants.AutoConstants.SLOW_SPEED, 3.0);
   }
+  
+  // needs testing 
+  /** robot drives onto charge station and balances */
+
+  public static CommandBase driveUpAndBalance(DriveSystem drivesystem) {
+    return Commands.sequence(
+
+      drivesystem.driveDistance(.5, 3),
+      drivesystem.autoBalance());
+      
+  }
+
+  /** robot stays still */
+
+  public static CommandBase doNothing(DriveSystem drivesystem) {
+    return Commands.sequence(drivesystem.driveDistance(0, 0));
+  }
```

Comment:
delete this and replace this option in the chooser with an empty instantcommand

do not use drivedistance here even with a distance and velocity value of 0 because a badly tuned pid controller could cause it to oscillate and still move